pkgs/applications/networking/instant-messengers/webex/version.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "42.8.0.22907",
+  "packageLocation": "https://binaries.webex.com/WebexDesktop-Ubuntu-Blue/20220712081040/Webex_ubuntu.7z",
+  "checksum": "b83950cdcf978a3beda93de27b25b70554fc82fcf072a5a7ea858d2ce0d176ac"
```

Comment:
I don't think we need this file. We can just update the information in the nix file.